recipe/meta.yaml

```diff
@@ -17,14 +17,23 @@ source:
   sha256: 090a84963ff15a8881a5f064c530c6e17a559968867ed4fdc3ac1f6d91047bc9  # [win]
 
 build:
-  number: 0
+  number: 1
+  skip: true  # [win and vc!=14]
+  msvc_compiler: 14.0  # [win]
```

Comment:
cribbed from https://github.com/conda-forge/geckodriver-feedstock

Reply:
This is pretty old syntax.  You should prefer this instead: https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html#anaconda-compilers-and-conda-build-3

Reply:
Errr... you have that.  This is superfluous.  The compiler stuff below will supersede anything that this might do.